eclipse-collections-code-generator/src/main/resources/impl/stream/primitiveSingletonSpliterator.stg

```diff
@@ -0,0 +1,69 @@
+import "copyrightAndOthers.stg"
+
+forSpecializedStream() ::= "true"
+
+targetPath() ::= "org/eclipse/collections/impl/stream"
```

Comment:
I think `stream/primitive` is more consistent with the rest of the packages. @motlin @donraab what do you think?